test/Graphics/RectangleShape.test.cpp

```diff
@@ -40,4 +40,10 @@ TEST_CASE("[Graphics] sf::RectangleShape")
         rectangle.setSize({5, 4});
         CHECK(rectangle.getSize() == sf::Vector2f(5, 4));
     }
+
+    SUBCASE("Geometric center")
+    {
+        sf::RectangleShape rectangle({20, 10});
```

Comment:
```suggestion
        const sf::RectangleShape rectangle({20, 10});
```
This ensures that `getGeometricCenter` is marked as `const`.